minidsp/board_2x4hd.py

```diff
@@ -101,14 +111,11 @@ def _setInputGain(self, input=0, gain=0):
         if (gain > 12) or (gain < -127.5):
             raise RuntimeError("Gain out of bounds. Range: -127.5 to 12 (db)")
 
-        # again, this is adapted from https://github.com/mrene/node-minidsp/
-        command = [0x13, 0x80, 0]
-        if input == 0:
-            command.append(0x1A)
-        elif input == 1:
-            command.append(0x1B)
-        else:
+        if input not in (0, 1):
```

Comment:
While `x in y` works for lists or tuples, it's more common to use lists. I'd prefer `if input not in [0, 1]:` but not a deal breaker.